teambuilder/web/views.py

```diff
@@ -72,6 +72,10 @@ def register(request):
     })
 
 
+def team_count():
+    return models.Team.object.all().count()
```

Comment:
* Such methods should be `staticmethod` to the class itself.
* You forgot **s** at the end of `object`.
* `.all()` is redundant.

In  `models.py` -
```python
class Team(models.Model):
    ...
    @staticmethod
    def count():
        return Team.objects.count()
    ...
```